src/Service/MoreInfoService/AbstractMoreInfoService.php

```diff
@@ -45,6 +45,9 @@
  */
 abstract class AbstractMoreInfoService extends SoapClient
 {
+    private const FALLBACK_CODE = 'fallback';
+    private const FALLBACK_IMAGE_URL = 'http://res.cloudinary.com/dandigbib/image/upload/v1543590683/default/forside-mangler.jpg';
```

Comment:
From environment variable instead?

Reply:
Agree in principle but we are being pragmatic in regards to this fix